app/components/start_page_banner/style.scss

```diff
@@ -1,13 +1,30 @@
 @import "../base.scss";
 @import "../../webpacker/styles/inverted";
 
+.app-start-page {
+  .app-govuk-phase-banner {
+    background-color: $govuk-brand-colour;
+    margin-top: -(govuk-spacing(2));
+  }
```

Comment:
Does this need to be nested under `.app-start-page`?

Reply:
I don't think you'd want govuk namespacing here - so `app-phase-banner` if it's just for us, or if it's about being inverted, `app-phase-banner--inverted`

Reply:
It needs to be nested under `.app-start-page` since that's the body class and this is the only page we're using this styling on.